wpa_supplicant/wpa_cli_zephyr.c

```diff
@@ -27,7 +27,7 @@
 #define CMD_BUF_LEN  1024
 #define MAX_CMD_SIZE 512
 #define MAX_RESPONSE_SIZE 512
-#define DEFAULT_IFNAME "wlan0"
+#define DEFAULT_IFNAME "nrf_wlan0"
```

Comment:
Please remove this setting, the default name should be ok here.

Reply:
Done.